common/src/main/java/co/elastic/otel/common/SpanValueStorageProvider.java

```diff
@@ -21,11 +21,27 @@
 import com.blogspot.mydailyjava.weaklockfree.WeakConcurrentMap;
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.sdk.trace.FieldBackedSpanValueStorageProvider;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import javax.annotation.Nullable;
 
 public interface SpanValueStorageProvider {
 
+  Logger logger = Logger.getLogger(SpanValueStorageProvider.class.getName());
+
   static SpanValueStorageProvider get() {
+    try {
```

Comment:
`FieldBackedSpanValueStorageProvider` directly references `SdkSpan`. If they are in different class loaders, this causes an `IllegalAccessError` here, which we didn't account for yet.